corehq/apps/userreports/templates/userreports/partials/link_reports.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+<button type="button"
+        data-toggle="modal"
+        data-target="#link-reports"
+        class="btn btn-default">
+  <i class="fa fa-link"></i> {% trans "Copy Report" %}
```

Comment:
Similar to the comment using "copy" in code, it'd be nice to explicitly reference linked domains/reports somewhere. I like the link icon but we don't use it anywhere else so I don't think it's a strong enough signal. Maybe the button should be "Link Report" instead of "Copy Report", what do you think?